src/heads-up/components/properties-item.vue

```diff
@@ -118,6 +151,12 @@ export default {
       }
       return null;
     },
+    warningMessage() {
+      if (this.warnings && this.warnings.length > 0) {
+        return this.warnings[0].message;
```

Comment:
Wat doe je met de andere warnings? Niet laten zien?

Reply:
Tot nu toe alleen de eerste, dit gaat in de toekomst nog veranderen